src/Title.js

```diff
@@ -0,0 +1,18 @@
+import React, {useState, useEffect} from 'react';
```

Comment:
안쓰고 계신건 삭제해주세요~~ㅋㅋㅋㅋ

```suggestion
import React from 'react';
```

Reply:
ㅋㅋㅋㅋㅋㅋㅋ 넵